io_scene_godot/converters/mesh.py

```diff
@@ -32,26 +32,21 @@ def export_mesh_node(escn_file, export_settings, node, parent_gd_node):
         return parent_gd_node
 
     else:
-        armature_data = armature.get_armature_data(node)
+        mesh_node = NodeTemplate(node.name, "MeshInstance", parent_gd_node)
+        mesh_exporter = MeshResourceExporter(node)
 
-        skeleton_node = None
+        armature_data = get_modifier_armature_data(node)
```

Comment:
Any reason you don't store the skeleton from the init_mesh_bones_data function?

Reply:
skeleton is stored in mesh node, not in the ArrayMesh resource, so I place it outside the class

Reply:
Ok, fair enough.